main/ertp/api/amount-math.md

```diff
@@ -1,189 +1,279 @@
+- `amount` `{Amount}`
 - Returns: `{Value}`
 
-Extract and return the value.
+Returns the `value` from the given `amount`.
 
 ```js
-const { amountMath } = makeIssuerKit('bucks');
-const fungible123 = amountMath.make(123);
+const { amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
+const quatloos123 = quatloosAmountMath.make(123);
 
 // returns 123
-const value = amountMath.getValue(amount);
+const myValue = quatloosAmountMath.getValue(quatloos123);
 ```
 
 ## amountMath.getEmpty()
 - Returns: `{Amount}`
 
-Return the amount representing an empty amount. This is the identity element for `MathHelpers.add()` and `MatHelpers.subtract()`.
+Returns the `amount` representing an empty `amount` for the `amountMath`'s 
+associated `brand`. This is the identity element for `AmountMath.add()` 
+and `AmountMath.subtract()`. The empty `value` depends on whether 
+the `amountMath` is of kind `nat` (`0`), `set` (`[]`), or `strSet` (`[]`).
 
 ```js
-const { amountMath } = makeIssuerKit('bucks');
-
-// Returns an empty amount for this issuer.
-// Since this is a fungible amount it returns 0
-const empty = amountMath.getEmpty();
+const { amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
+// Returns an empty amount for this amountMath.
+// Since this is a fungible amount it returns an amount
+// with 0 as its value.
+const empty = quatloosAmountMath.getEmpty();
 ```
 
 ## amountMath.isEmpty(amount)
 - `amount` `{Amount}`
-- Returns: `{boolean}`
+- Returns: `{Boolean}`
 
-Return true if the amount is empty. Otherwise false.
+Returns `true` if the `amount` is empty. Otherwise returns `false`.
 
 ```js
-const { amountMath } = makeIssuerKit('fungible');
-const empty = amountMath.getEmpty();
-const fungible1 = amountMath.make(1);
+const { amountMath: quatloosAmountMath } = makeIssuerKit('quatloos');
+const empty = quatloosAmountMath.getEmpty();
+const quatloos1` = quatloosAmountMath.make(1);
 
 // returns true
-amountMath.isEmpty(empty)
+quatloosAmountMath.isEmpty(empty)
 
 // returns false
-amountMath.isEmpty(fungible1)
+quatloosAmountMath.isEmpty(quatloos1)
 ```
 
 ## amountMath.isGTE(leftAmount, rightAmount)
 - `leftAmount` `{Amount}`
 - `rightAmount` `{Amount}`
 - Returns: `{boolean}`
 
-Returns true if the leftAmount is greater than or equal to the rightAmount. For non-scalars, "greater than or equal to" depends on the kind of amount, as defined by the MathHelpers. For example, whether rectangle A is greater than rectangle B depends on whether rectangle A includes rectangle B as defined by the logic in MathHelpers.
+Returns `true` if the `value` of `leftAmount` is greater than or equal to
+the `value` of `rightAmount`. Both `amount` arguments must have the same
+`brand`.
+
+For non-fungible `values`, what "greater than or equal to" is depends on the 
+kind of `amountMath`. For example, whether rectangle A is greater than rectangle B 
+depends on whether rectangle A includes rectangle B as defined by the logic in `amountMath`
```

Comment:
How's this?

For example, { 'seat 1', 'seat 2' } is considered
greater than { 'seat 2' } because the former 
both contains all of the latter's  contents and has more elements.

Reply:
that's good.